### PR TITLE
Katello 3.14 RC1 documentation

### PR DESCRIPTION
**Pull request by jturel**
This PR depends on (and includes) https://github.com/theforeman/theforeman.org/pull/1494 so they can go together or separately (I'll rebase it out if the latter)

### CODE REVIEW

File: plugins/katello/3.14/installation/index.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  <div class="alert alert-danger">
+    These are the instructions for installing the unstable nightly release of Katello!
+  </div>
+{% elsif page.version != page.latest %}
+  <div class="alert alert-danger">
+    These instructions are for installing Katello {{ page.version }}, but the latest stable is <a href="/plugins/katello/{{ page.latest }}/installation/index.html">{{ page.latest }}</a>.
+  </div>
+{% endif %}
+
+Note: After installation of Katello, be sure to trust Katello's CA certificate
```
Comment (swadeley) [inline plugins/katello/3.14/installation/index.md]: Note: After installation of Katello, navigate to the `/pub` directory and trust the Katello's CA certificate for identifying web sites (e.g. `http://katello.example.com/pub/katello-server-ca.crt`).

Comment (jturel) [reply plugins/katello/3.14/installation/index.md]: Is the distinction here that you want to omit the 'note' part?

Comment (swadeley) [reply plugins/katello/3.14/installation/index.md]: No, sorry. My  intention was to suggest you replace all of the text after Note: with the text I have proposed. The important thing is to remove "be sure to trust" as its better to just tell people what to do, and then the "You will find" is just too wordy. 

Adding "This is required for the encrypted NoVNC connections." is OK, but would be best left to a procedure on NoVNC as trusting the cert is also required for other reasons.

Comment (swadeley) [reply plugins/katello/3.14/installation/index.md]: @jturel I added the Note: back.

Comment (jturel) [reply plugins/katello/3.14/installation/index.md]: Updated. I took the liberty of omitting 'Note' and surrounding the message with an 'info' CSS style to make appropriately bring attention to it

**Comment by jjeffers** [comment]
Looks spiffy to me. I appreciate the DRYness.

**Comment by jturel** [comment]
> Looks spiffy to me. I appreciate the DRYness.

Thanks! Good enough for a formal approval?

File: plugins/katello/3.14/advanced/certificates.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+If you would like to configure Katello with a set of invalid certs, 
+the validation check can be skipped by passing `--certs-skip-check` to the installer.
+
+## Custom Server Certificates
+### New Katello Installations
+
+```
+foreman-installer --scenario katello\
+                  --certs-server-cert ~/path/to/server.crt\
+                  --certs-server-cert-req ~/path/to/server.crt.req\
```
Comment (ekohl) [inline plugins/katello/3.14/advanced/certificates.md]: This is optional and we should stop advising its use since it's not needed.

**Comment by jjeffers** [comment]
> > Looks spiffy to me. I appreciate the DRYness.
> 
> Thanks! Good enough for a formal approval?

IMO, yes, but I don't have approval rights, I don't think.

File: plugins/katello/3.14/advanced/certificates.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+server certs. This CA gets distributed to content hosts and Smart Proxies.
+
+For Smart Proxies the following options are passed to
+`foreman-proxy-certs-generate`:
+
+```
+foreman-proxy-certs-generate --foreman-proxy-fqdn "$FOREMAN_PROXY"\
+                             --certs-tar ~/$FOREMAN_PROXY-certs.tar\
+                             --server-cert ~/path/to/server.crt\
+                             --server-cert-req ~/path/to/server.crt.req\
```
Comment (ekohl) [inline plugins/katello/3.14/advanced/certificates.md]: Same here.

File: plugins/katello/3.14/upgrade/clients.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+Update the Katello client release packages:
+
+<p>
+  Select your Operating System:
+  <select id="operatingSystems">
+     <option value="el5">Enterprise Linux 5 (RHEL, CentOS, etc.)</option>
+     <option value="el6">Enterprise Linux 6 (RHEL, CentOS, etc.)</option>
+     <option value="el7">Enterprise Linux 7 (RHEL, CentOS, etc.)</option>
+     <option value="f27">Fedora 27</option>
```
Comment (ekohl) [inline plugins/katello/3.14/upgrade/clients.md]: We don't support Fedora 27 anymore

**Comment by jturel** [comment]
@ekohl  thanks for the feedback - updated docs in the areas you mentioned

**Comment by jturel** [comment]
@ekohl ACK?

**Review by ekohl — APPROVED** [review]
